core/camel-endpointdsl/src/main/java/org/apache/camel/builder/endpoint/dsl/ServiceNowEndpointBuilderFactory.java

```diff
@@ -1034,7 +1034,7 @@ default AdvancedServiceNowEndpointBuilder timeFormat(String timeFormat) {
 
     public interface ServiceNowBuilders {
         /**
-         * ServiceNow (camel-servicenow)
+         * ServiceNow (@@@ARTIFACTID@@@)
```

Comment:
This is really weird.

Reply:
I think is something else, not from the endpointdsl MOJO, looks like the generated `json` for this component has this `@@@ARTIFACTID@@@`

Reply:
Did you rebuild everything without fastinstall and -DskipTests?

Reply:
Nope I built it with fastinstall, could be that?

Reply:
There were a lot of changes yesterday so I would go with a full rebuild in this case.